tensorflow/lite/kernels/kernel_util.h

```diff
@@ -119,6 +119,20 @@ TfLiteStatus GetTemporarySafe(const TfLiteContext* context,
                               const TfLiteNode* node, int index,
                               TfLiteTensor** tensor);
 
+// Same as `GetScratch` but returns boolean and uses output argument for
+// tensor.
+//
+//   TfLiteTensor* my_tensor;
+//   TF_LITE_ENSURE_OK(context,
+//                     GetTemporarySafe(context, node, kMyTensorIdx,
+//                     &my_tensor));
+//   // can use my_tensor directly from here onwards, it is not nullptr
+//
+// Should be used in cases where the binary size is too large.
+TfLiteStatus GetScratchSafe(const TfLiteContext* context,
+                            const TfLiteNode* node, int index,
+                            TfLiteTensor** tensor);
```

Comment:
Is this change required, or just a nice-to-have?

Reply:
this API is required to get scratch buffer at prepare stage of LSTM op